### PR TITLE
Update README for release

### PR DESCRIPTION
**Pull request by TommyHuang-dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,13 @@
...
-
-## Running end-to-end tests
-
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
-
-## Further help
-
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+## Our Goal
+The goal of virtrolio is to allow users to write memorable messages to one another. We hope to give everyone the chance to do this, even if they cannot physically send anything.
```
Comment (GuyInFridge) [inline README.md:13]:
```suggestion
The goal of virtrolio is to allow users to write memorable messages to one another. We hope to give everyone the chance to do this, even if they cannot physically be together.
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,13 @@
-# VirtrolioSite
+<img align="left" src="https://github.com/virtrolio/virtrolio-site/blob/master/src/assets/images/logo_reg.png" width="200px" height="200px">
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.1.6.
+# Virtrolio
+Virtrolio is a webservice that lets you send messages to a friend or colleague's virtual book. It was originally created due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Link to the website: https://virtrolio.web.app/
```
Comment (GuyInFridge) [inline README.md:4]:
```suggestion
Virtrolio was founded due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Virtrolio is a web service that lets you privately send messages to a friend or colleague and allows only them to view the messages that are destined for them. With the help of some HTML, CSS, Typescript, love, sweat, and tears, we've combined the fun, privacy, and intimacy of yearbook signing with online features such as image embedding and more! 

Check it out! All you need is a Google Account: https://virtrolio.web.app/
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,13 @@
...
 
-## Development server
+<br>
+<br>
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+## How We Built It
+Virtrolio is built using Angular and Bootstrap in the frontend, with Firebase for data storage, hosting, and user authentication. Firebase was chosen for it's ease of use and integration with various login systems. By using well-tested login systems we remove the need to worry about login security.
```
Comment (GuyInFridge) [inline README.md:10]:
```suggestion
Virtrolio is built using Angular and Bootstrap in the front end, with Firebase for data storage, hosting, and user authentication. Firebase was chosen for its ease of use and integration of various secure login authentication providers.
```

File: README.md (hunk 2 above)
Comment (janakitti) [inline README.md:4]:
```suggestion
Virtrolio is a webservice that lets you send messages to a friend' or colleague's virtual yearbook. It was originally created due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Link to the website: https://virtrolio.web.app/
```

Comment (ricetech) [reply README.md:4]: The link to the website should be at the very top, before the first section.

Comment (TommyHuang-dev) [reply README.md:4]: I think it makes more sense in context if it comes afterwards

**Pull request merged**